docs/user_guide/PXE_Boot.inc

```diff
@@ -201,9 +201,17 @@ various files to model different systems.
      kickstart file needed.
 
      .. NOTE::
-        If using the default cfg files, know that they do not have the '_el[6,7]' 
+        If using the default cfg files, know that they do not have the '_el[6,7]'
         tags at the end of their name.
```

Comment:
Should be "names" to follow plurality of the rest of the sentence?

Reply:
@sharkbruhaha If you feel that minor syntactical changes are necessary, can you suggest them via the GitHub editor?